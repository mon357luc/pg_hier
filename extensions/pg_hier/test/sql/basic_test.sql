-- Basic test for pg_hier extension
-- Test extension creation and basic functionality

-- Create the extension
CREATE EXTENSION IF NOT EXISTS pg_hier;

-- Test basic extension installation
SELECT extname FROM pg_extension WHERE extname = 'pg_hier';

-- Test that extension is properly loaded
\dx pg_hier

-- Test extension version
SELECT extversion FROM pg_extension WHERE extname = 'pg_hier';

-- Clean up
DROP EXTENSION pg_hier;
