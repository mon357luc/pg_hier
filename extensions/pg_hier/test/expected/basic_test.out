-- Basic test for pg_hier extension
-- Test extension creation and basic functionality
-- Create the extension
CREATE EXTENSION IF NOT EXISTS pg_hier;
-- Test basic extension installation
SELECT extname FROM pg_extension WHERE extname = 'pg_hier';
 extname 
---------
 pg_hier
(1 row)

-- Test that extension is properly loaded
\dx pg_hier
                                    List of installed extensions
  Name   | Version | Schema |                              Description                              
---------+---------+--------+-----------------------------------------------------------------------
 pg_hier | 0.1     | public | pg_hier makes it less verbose to query hierarchical structured tables
(1 row)

-- Test extension version
SELECT extversion FROM pg_extension WHERE extname = 'pg_hier';
 extversion 
------------
 0.1
(1 row)

-- Clean up
DROP EXTENSION pg_hier;
